tracer/build/_build/Build.GitHub.cs

```diff
@@ -804,6 +792,42 @@ await using (Stream file = File.Create(zipPath))
         Console.WriteLine($"Artifact download complete");
     }
 
+    static async Task DownloadGitlabArtifacts(AbsolutePath outputDirectory, string commitSha, string version)
+    {
+        var awsUri = $"https://dd-windowsfilter.s3.amazonaws.com/builds/tracer/{commitSha}/";
+        var artifactsFiles= new [] 
+        {
+            $"{awsUri}x64/en-us/datadog-dotnet-apm-{version}-x64.msi",
+            $"{awsUri}x86/en-us/datadog-dotnet-apm-{version}-x86.msi", 
+            $"{awsUri}windows-native-symbols.zip"
+        };
+
+        foreach (var fileToDownload in artifactsFiles)
+        {
+            var fileName = fileToDownload.Substring(fileToDownload.LastIndexOf('/') + 1);
+            var destination = outputDirectory / commitSha / fileName;
+            EnsureExistingDirectory(destination);
+
+            Console.WriteLine($"Downloading {fileName} to {destination}...");
+
+            using (var client = new HttpClient())
```

Comment:
`HttpClient` is designed to be reused, so I would suggest creating it outside of the `foreach`.